runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePlugin.java

```diff
@@ -55,7 +55,7 @@
 	@Override
 	protected void startUp() throws Exception
 	{
-		navButton = new NavigationButton("Hiscore", () -> hiscorePanel);
+		navButton = new NavigationButton("HiscorePlugin", () -> hiscorePanel);
```

Comment:
This is the tooltip for the nav button and should not be renamed

Reply:
Fixed with fa384b4